packages/malloy-interfaces/thrift/malloy.thrift

```diff
@@ -277,10 +246,12 @@ struct Field {
   1: required Expression expression,
   // TODO only two kinds of distinguishable annotations are before `aggregate:` and before `name is value`
   // between `name` and `is`, or between `is` and `value` are converted to before `name`.
+  // TODO consider having an annotation tree with inheritance
```

Comment:
Do we want to have the full annotation tree, rather than just the flattened list? This tree structure is currently not used by anyone, but might be useful for people.

Reply:
Michael's vote is to keep the tree; it's easy to flatten it as a user.